src/components/GeographyMap/index.tsx

```diff
@@ -0,0 +1,108 @@
+import { MapContainer, GeoJSON, TileLayer } from 'react-leaflet';
+import { LatLng, geoJSON as createGeoJSON, latLng } from "leaflet";
+import 'leaflet/dist/leaflet.css';
+import { Overlay, Paper } from '@mantine/core';
+import { parseValue } from "~/util/surrealql";
+import { GeometryCollection, GeometryLine, GeometryMultiLine, GeometryMultiPoint, GeometryMultiPolygon, GeometryPoint, GeometryPolygon } from "surrealdb.js";
+import { useIsLight } from '~/hooks/theme';
+import { useEffect, useState } from 'react';
+
+export type GeographyInput =
+	| GeometryPoint
+	| GeometryLine
+	| GeometryPolygon
+	| GeometryMultiPoint
+	| GeometryMultiLine
+	| GeometryMultiPolygon
+	| GeometryCollection<any>;
+
+const convertCoordsToLatLng = (point: [number, number] | [number, number, number]): LatLng => {
+	return latLng({ lat: point[1], lng: point[0] });
```

Comment:
Somwhere is bug...
Here: long, lat
<img width="957" alt="image" src="

">

And here: lat, long.
https://surrealdb.com/docs/sdk/php/data-types
<img width="791" alt="image" src="https://github.com/user-attachments/assets/76e6f133-43cd-451f-8d28-f3c6d483ed1f">


So at the end... adding `new GeometryPoint([lat, long])` to database, will incorrectly display data in this app